infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/TrackLayout.kt

```diff
@@ -142,6 +142,7 @@ data class LocationTrackInfoboxExtras(
     val duplicates: List<LocationTrackDuplicate>,
     val switchAtStart: LayoutSwitchIdAndName?,
     val switchAtEnd: LayoutSwitchIdAndName?,
+    val partOfUnfinishedSplit: Boolean?,
```

Comment:
Päädyin lisäämään tämän tänne InfoboxExtras-luokkaan aika lailla sen takia, että muuten joko `LocationTrack`:iin olisi pitänyt tuoda näin laskettua tietoa, tai sitten olisi joutunut suorittamaan useamman haun -> ajattelin että tämä on pienin paha

Reply:
True, ja jos niillä on ero, kannattaisi oikeastaan harkita enumia. 3-tila-booleanit kommunikoivat kovin heikosti että miksi niillä on 3 tilaa.